### ISSUE TITLE
Cannot check package versions in Teamwork

### ISSUE THREAD

**Issue by rzadp** [dxos/version-check#3]
To reproduce cleanly from scratch:

```
git clone https://github.com/dxos/teamwork.git && cd teamwork
yarn add -DW @dxos/version-check
yarn version-check upgrade --scope @dxos
```

Result:

```
16:12 ➜ yarn version-check upgrade --scope @dxos
yarn run v1.22.5
$ /Users/przemek/Projects/temp/teamwork/node_modules/.bin/version-check upgrade --scope @dxos
Querying NPM for package manifests...
(node:60893) UnhandledPromiseRejectionWarning: Error: Failed to get package manifest: Not found
    at Object.getPackageManifest (/Users/przemek/Projects/temp/teamwork/node_modules/@dxos/version-check/dist/src/npm.js:15:15)
    at processTicksAndRejections (internal/process/task_queues.js:94:5)
    at async /Users/przemek/Projects/temp/teamwork/node_modules/@dxos/version-check/dist/src/commands/upgrade.js:31:27
    at async Promise.all (index 2)
    at async Object.upgrade (/Users/przemek/Projects/temp/teamwork/node_modules/@dxos/version-check/dist/src/commands/upgrade.js:30:5)
(node:60893) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). (rejection id: 1)
(node:60893) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.
✨  Done in 4.54s.
```

**Comment by rzadp** [comment]
Also another issue is that it exits with positive exit code:

```
16:12 ➜ echo $?
0
```

**Comment by rzadp** [comment]
Fixed in `0.1.9`
